common/src/main/java/com/lightstep/tracer/shared/Util.java

```diff
@@ -53,4 +54,12 @@ static long fromHexString(String hexString) {
     static String toHexString(long l) {
         return Long.toHexString(l);
     }
+
+    static boolean IsNotMetaSpan(Span span) {
+        if (span == null) {
+            return true;
+        }
+        KeyValue kvp = KeyValue.newBuilder().setKey("lightstep.meta_event").setBoolValue(true).build();
```

Comment:
(I'm also curious about how expensive this part is - hopefully it's not ;) )

Reply:
I was curious about that as well, but I didn't see another handy way to get tags off of a span. Do you have any thoughts?